Automattic/jetpack
# Debug Helper Plugin: add a Sync Data Settings Tester page

#### Changes proposed in this Pull Request:
* It would be useful to have an easy way to observe the Sync data settings provided by the Sync data filters. Let's add a Sync Data Settings Tester page to the Debug Helper plugin. The output of the `apply_filters` call for each Sync data filter will be displayed on this page.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. Install and activate this branch of the Debug Helper plugin.
2. Install, activate, and connect Jetpack.
3. On the Debug Helper dashboard, enable the Sync Data Setting Tester module.
4. Navigate to the Sync Data Setting Tester dashboard.
5. Observe the Sync data settings.
